docs/flask/1/authorization-server.rst

```diff
@@ -222,12 +222,14 @@ request::
 
         granted = request.form.get('granted')
         if granted:
-            grant_user = current_user
+            client_id = request.form.get('client_id')
+            client = server.get_client_by_id(client_id)
+            grant_user = client.user
```

Comment:
grant_user should be `current_user`. This `current_user` is defined by developers.

Reply:
Got it, I just found out the current_user implementation in the playground project, thanks!